### PR TITLE
Add entry about no longer asking for a trainee’s home address

### PR DESCRIPTION
**Pull request by johndoates**

### CODE REVIEW

File: app/posts/register-trainee-teachers/2023-05-22-no-longer-asking-for-a-trainees-home-address.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: No longer asking for a trainee’s home address
+description: We’ve stopped importing, collecting or displaying home addresses.
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2023-05-22-no-longer-asking-for-a-trainees-home-address.md]:
```suggestion
description: We’ve stopped importing, collecting and displaying home addresses.
```

File: app/posts/register-trainee-teachers/2023-05-22-no-longer-asking-for-a-trainees-home-address.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+date: 2023-05-19
+---
+
+We currently collect a trainee’s home address when:
+
+- we import their data from the Manage teacher training applications service
+- a user manually adds their data to Register
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2023-05-22-no-longer-asking-for-a-trainees-home-address.md]:
```suggestion
- a user manually adds their trainees to Register
```

File: app/posts/register-trainee-teachers/2023-05-22-no-longer-asking-for-a-trainees-home-address.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+The Higher Education Statistics Authority (HESA) does not collect home addresses. This means that we do not have home addresses for trainees who are training with higher education institutions.
+
+The DfE no longer needs to write to trainees at their home addresses. We use email to send messages to them, for example when they’ve been awarded teaching status.
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2023-05-22-no-longer-asking-for-a-trainees-home-address.md]:
```suggestion
The DfE no longer needs to write to trainees at their home addresses. We now use email to send messages to them, for example when they’ve been awarded teaching status.
```

File: app/posts/register-trainee-teachers/2023-05-22-no-longer-asking-for-a-trainees-home-address.md (hunk 2 above)
Comment (johndoates) [reply app/posts/register-trainee-teachers/2023-05-22-no-longer-asking-for-a-trainees-home-address.md]: The intention here was for "their" in the bullets to refer to the trainee. So it's the trainee's data. However if that isn't clear I'll make a change.

**Review by edwardhorsford — APPROVED** [review]

**Pull request merged**